coding-concepts.js

```diff
@@ -16,44 +16,49 @@ const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2022"
-// console.log(cohort.length)
+console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: it will come back as number 10
+// b) Verify and explain: the .lenght counts the number of characters inside the string, learn is 5 and 2022 is 4 but the space in between those counts as a character so in total is 10.
 
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: I belive it will show one of the letters, the letter "o" of Hello
+// b) Verify and explain: I was correct, it returned the letter "o" I know that in coding when we search for the placing of a letter it starts with 0, so if we start counting from zero in the "Hello world" the letter o is the fourth.
 
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
-// console.log(languages[index])
+console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: I think that it will show the either the first letter of the first word or the first word "JavaScript" but the index 1 is throwing me off a little
+// b) Verify and explain: It showed "Ruby", because when the index was added it was looking for the first value (Ruby) so because the string is zeroth indexed the 0 is "JavaScript"
```

Comment:
yes to research. Trickery because the word `index` was used to hold the number 1. So the question is asking for the value at the index of 1.